tests/Unit/Crypt/RSA/ModeTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * @author    Jim Wigginton <terrafrost@php.net>
+ * @copyright 2013 Jim Wigginton
+ * @license   http://www.opensource.org/licenses/mit-license.html  MIT License
+ */
+
+require_once 'Crypt/RSA.php' ;
+
+class Unit_Crypt_RSA_ModeTest extends PhpseclibTestCase
+{
+    public function testEncryptionModeNone()
+    {
+        $plaintext = 'a';
+
+        $rsa = new Crypt_RSA();
+
+        extract($rsa->createKey());
+        $rsa->loadKey($publickey);
+
+        $rsa->setEncryptionMode(CRYPT_RSA_ENCRYPTION_NONE);
+        $a = $rsa->encrypt($plaintext);
+        $b = $rsa->encrypt($plaintext);
+
+        $this->assertEquals($a, $b);
```

Comment:
In order to do that the key can't be random, which it currently is. I can hard-code the key tho. I guess I'll do that in another commit shortly.